.github/workflows/ci.yaml

```diff
@@ -1,8 +1,6 @@
 name: CI
 on:
   push:
-    branches:
```

Comment:
uhh why?

Reply:
As my branch became longer-living than expected, I wanted to run the build on my branch too. Happy to roll back if you want to keep it in main only.

Reply:
I guess also because the actions doesn't run automatically on this PR. But I also often have several branches that I'd like to have checked before creating the PR or while testing random stuff.

Reply:
Please add this back so merges to `main` will trigger CI runs.

Reply:
I'm a bit surprised it is not building on main, it is building on my branch: https://github.com/trygvis/ractor_actors/actions/runs/14497493422/job/40668831660.

I've put it back, but for later it would be nice to get both branches to build.